tagblock.go

```diff
@@ -0,0 +1,111 @@
+package nmea
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+const (
+	// TypeUnixTime unix timestamp, parameter: -c
+	TypeUnixTime = "c"
+	// TypeDestinationID destination identification 15char max, parameter: -d
+	TypeDestinationID = "d"
+	// TypeGrouping sentence grouping, parameter: -g
+	TypeGrouping = "g"
+	// TypeLineCount linecount, parameter: -n
+	TypeLineCount = "n"
+	// TypeRelativeTime relative time time, paremeter: -r
+	TypeRelativeTime = "r"
+	// TypeSourceID source identification 15char max, paremter: -s
+	TypeSourceID = "s"
+	// TypeTextString valid character string, parameter -t
+	TypeTextString = "t"
+)
+
+var (
+	// TagBlockRegexp matches nmea tag blocks
+	TagBlockRegexp = regexp.MustCompile(`^(.*)\\(\S+)\\(.*)`)
+)
+
+// TagBlock struct
+type TagBlock struct {
+	Head         string // *
+	Time         uint32 // -c
```

Comment:
We stick to `int64` or `float64` for the numeric types in the codebase.

Reply:
fixed in latest commit